yarn.lock

```diff
@@ -8757,6 +8757,11 @@ lru-cache@^6.0.0:
   dependencies:
     yallist "^4.0.0"
 
+macaddress@^0.5.3:
```

Comment:
Have we considered generating a unique id in the environment and persists across runs over using PII information?

Reply:
@shauns in the spreadsheet the note about this field indicates `MAC address (in theory unique) of the machine executing the command`, do we only need the `macaddress` as a unique identifier of the CLI instance?, if that is the case, I agree with @pepicrft about generating a unique id. The problem would be that in case the environment variables are cleared, the relationship with previous execution will be lost

Reply:
Finally the mac address fields has been removed and added a new one `env_device_id` instead. The value of this field will be the mac address hash to avoid the creation of a new unique id that should be stored across executions